hack/local-imagevector-overwrite.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
```

Comment:
Instead of copying this file here, can you rather keep it in g/g and make re-usable? Other extensions might need the same in the future as well.

Reply:
Will work on that an rebase it here as soon as it is in g/g

Reply:
Updated the PR accordingly.